chore(blog): add the 2024 annual report (#3578)
asyncapi/website: markdown/blog/2024-annual-summary.md

If you are curious about our past achievements, [please read the previous Project Status updates](https://www.asyncapi.com/blog?tags=Project+Status).

## Slack

We currently have a total of **6,124** members in the AsyncAPI workspace. Slack is our primary communication channel for sharing updates, assisting contributors and adopters who want to understand and use the AsyncAPI specification and toolings.

We welcomed **1,505** new members in 2024.

<Figure
  src="/img/posts/2024-annual-report/new-slack-users.webp"
  caption="New members joining Slack each year"
  className="text-center"
/>

Among this year's new joiners, we have our top five members, including our community marketing specialist, who leads the marketing efforts in the community.

<Figure
  src="/img/posts/2024-annual-report/slack-top-five.webp"
  caption="Top 5 new members on Slack"
  className="text-center"
/>

Our weekly activity slightly dropped this year due to changes that affected some of our core maintainers after the layoffs. 

<Figure
  src="/img/posts/2024-annual-report/weekly-slack-activity.webp"
  caption="Slack weekly activity over the past years"
  className="text-center"
/>

> ## Social Media 

Reviewer: ```suggestion
## Social Media Presence
```

> ## Social Media 
> Our social media platforms help us communicate community news, share updates, and make announcements.
> Let's dive into the numbers:

Reviewer 2: ```suggestion
## Social Media Growth and Impact
Our social media platforms are key channels for communicating community news, sharing updates, and making announcements. Here's a detailed breakdown of our social media metrics for 2024:
```

Author: I'll rather keep the original version Akshat

<Figure
  src="/img/posts/2024-annual-report/linkedin-followers.webp"
  caption="Yearly followers on LinkedIn"
  className="text-center"
/>

Compared to 2023, the number of visits from users and new users to [asyncapi.com](https://www.asyncapi.com), referred by LinkedIn, decreased by 40% in 2024. However, on a positive note, we received a significant increase in sessions, which kept users more engaged.

<Figure
  src="/img/posts/2024-annual-report/visits-from-linkedin.webp"
  caption="Website visits referred by LinkedIn"
  className="text-center"
/>

We don't use any premium or paid marketing services!

### X/Twitter
We gained **309** followers in 2024, reaching a total of **4709** followers overall. Website referrals declined 42% compared to the previous years, but sessions on the website increased significantly.

<Figure
  src="/img/posts/2024-annual-report/visits-from-twitter.webp"
  caption="Website visits referred by Twitter"
  className="text-center"
/>

We don't consider the X/Twitter platform worth investment; it's time to consider another platform like Bluesky to expand our reach.

### YouTube

> Our YouTube channel continues to grow as we gain **269** new subscribers, having **1.72k** subscribers in total. We hosted fewer community live streams in 2024 but also brought back the AsyncAPI Online Conference, increasing the number of watch time.

Reviewer: ```suggestion
Our YouTube channel continues to grow as we gain **269** new subscribers, having **1.72k** subscribers in total. We hosted fewer community live streams in 2024 but also brought back the AsyncAPI Online Conference, increasing the total watch time.
```

<Figure
  src="/img/posts/2024-annual-report/issues.webp"
  caption="Total number of issues closed"
  className="text-center"
/>

### NPM
The most controversial part of the review is the number of downloads from AsyncAPI's top packages, which are hosted under the Node Package Manager. The `asyncapi/specs` package continues to soar, with over **34M** downloads in the past year. 

<Figure
  src="/img/posts/2024-annual-report/npm-packages.webp"
  caption="Total downloads of AsyncAPI top packages"
  className="text-center"
/>

## Community Achievements
Despite the challenges, we achieved some milestones, and here are some of the highlights from the AsyncAPI community in 2024!

- We had our second community hire, [Oluwabamikemi Kayode](https://www.linkedin.com/in/atinuke-oluwabamikemi-kayode-5b838b1b7/), to work full-time as the project's Community Marketing Specialist.
- The [Ambassador Program](https://www.asyncapi.com/community/ambassadors) increased this year from having five ambassadors to ten.

> - The [AsyncAPI Bounty Program](https://www.asyncapi.com/blog/asyncapi-bounty-program-2024) turned 1.

Reviewer 3: can you provide just an overall number on how many issues were completed
